### PR TITLE
Feature/export metrics

### PR DESCRIPTION
**Pull request by Nainterceptor**
Checklist

- [x] I've added a mention in CHANGELOG.md

### CODE REVIEW

**Comment by Nainterceptor** [comment]
[Metrics / Logs dashboard](https://trello.com/c/fWgY8UGA/297-metrics-logs-dashboard)

File: .helm-chart/templates/prometheus-service-monitor.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+metadata:
+  name: {{ $fullName }}
+  labels:
+    app: {{ $name }}
+    heritage: {{ .Release.Service }}
+    release: prometheus
+spec:
+  endpoints:
+    - interval: 30s
+      port: metrics
```
Comment (Scttpr) [inline .helm-chart/templates/prometheus-service-monitor.yaml]: Is this a variable ? If not, you don't need to have a number here ?

File: Dockerfile [hunk 2]
```diff
@@ -22,6 +22,7 @@ COPY . /usr/src/app/
 RUN npm run build
 
 EXPOSE 3000
+EXPOSE 9091
```
Comment (Scttpr) [inline Dockerfile]: Don't you override line 24 here ?

File: .helm-chart/templates/prometheus-service-monitor.yaml (hunk 1 above)
Comment (Nainterceptor) [reply .helm-chart/templates/prometheus-service-monitor.yaml]: Like a variable, it will refer to the name, in service yaml

File: Dockerfile (hunk 2 above)
Comment (Nainterceptor) [reply Dockerfile]: No, in docker, it will just expose both ports ([see documentation](https://docs.docker.com/engine/reference/builder/#expose))

**Pull request merged**